docs/openapi-user.yaml

```diff
@@ -93,183 +109,50 @@ paths:
           $ref: "#/components/responses/Unexpected"
 
 components:
+  securitySchemes:
```

Comment:
must match this one: https://github.com/pagopa/io-functions-app/blob/master/openapi/index.yaml#L354

(header name is `X-Functions-Key`)